testing/test_assertrewrite.py

```diff
@@ -1371,10 +1371,21 @@ def test_fails():
         )
         result = testdir.runpytest()
         result.stdout.fnmatch_lines(
-            "*Assertion Passed: a + b == c + d (1 + 2) == (3 + 0) at line 7*"
+            "*Assertion Passed: a+b == c+d (1 + 2) == (3 + 0) at line 7*"
```

Comment:
Did you see the approach I took in https://github.com/pytest-dev/pytest/pull/5511 of using `inline_run` and checking the parameters received by the hook directly? Feels more elegant that way as we can check the parameters directly instead of relying on parsed output.

Reply:
I did yeah, didn't quite feel like expanding the api scope of pytester further but I can take another look at that again :+1:

Reply:
I added a two-liner helper function to `ParsedCall`, that's all. I think it is an improvement.

Anyway just a suggestion really. 👍